README.md

```diff
@@ -78,6 +82,37 @@ $ ../build/nothing
 > ..\build\nothing
 ```
 
+### MinGW (with MSYS)
```

Comment:
Same situation here. Windows should have three subsections: "Visual Studio", "MinGW (with MSYS)" and "MinGW (without MSYS)"